static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -53,7 +53,7 @@ example, if a data file `Posts.xml.zip` has checksum
 `ec1d2935f811b77cc49b031b999cbf17`, its cache entry will be
 `.dvc/cache/ec/1d2935f811b77cc49b031b999cbf17` locally.
 
-> **Note!** File checksums are calculated from file contents only. 2 or more
```

Comment:
a lamp emoji here? or maybe a note emoji?

Reply:
Idk. It's just a note about internal implementation (BTW related to #68), not a warning of some sort. So far we're using lamp for external tool tips. I'd prefer to leave this for a more extense review of all notes when addressing #855. I updated that issue's description BTW

Reply:
👍